.env.example

```diff
@@ -12,14 +12,14 @@ LOG_LEVEL=debug
 CURRENT_UID=1000
 
 DB_CONNECTION=mysql
-DB_HOST=intern-db-dev
-DB_PORT=5432
+DB_HOST=intern-mysql
+DB_PORT=3306
 DB_DATABASE=intern
 DB_USERNAME=intern
 DB_PASSWORD=password
 DB_ROOT_PASSWORD=example
 
-DOCKER_DEV_DB_EXTERNAL_PORT=5432
+DOCKER_DEV_DB_EXTERNAL_PORT=3306
```

Comment:
Please provide information in readme that you're blocking default MySQL port by default. The instruction should contain a hint how to change this external port as this probably will be a common problem.